### ISSUE TITLE
General question - what is this project for?

### ISSUE THREAD

**Issue by lopezwys** [Fuzzy-Mind/WirelessPhototherapy#1]
Hi, i've only recently got into open source and programming in general. I have very limited experience in coding, so i figured it would be easier to ask you directly rather than trying to reverse engineer this repo contents. Yesterday i read about a novel approach to phototherapy - devices which flash light directly into ear canal, which potentially would be more efficient in curing SAD symptoms than the traditional phototherapy lights. I think that it should be relatively easy to build this kind of device at home and (correct me if i'm wrong) with an appropriate build&usage guide it should also be quite safe. Your repo has caught my attention, but unfortunately i couldn't find any kind of a readme file. That said - what is this project for? If it isn't for above-mentioned personal devices, do you think it would be difficult to develop something like this?

PS. I don't intend to build in-ear phototherapy lights, i'm just curious about the topic, because if i'm correct, that kind of a project could benefit many people. Especially if it was open-source

**Comment by Fuzzy-Mind** [comment]
Hello my friend,

I work as an R&D engineer on medical device company. I am developing
hardware and software for Neonatal intensive care units.
This repository has been uploaded for archiving purposes only so it is not
an open source project actually.
This project belongs to the phototherapy device used for the treatment of
jaundice in newborns. You can see on this link
<https://images.app.goo.gl/Ft3HzSYT6Q16P5SS7>.
I examined briefly the phototherapy device used for the treatment of SAD
via ear canal you mentioned and it really impressed me.
In my opinion the project can be done at home as open source with simple
hardware and software.
But there are some questions and critical points that need to be answered.

1- How many mw/cm2 should the LED power be?
2- What should be the distance between the light source and the ear?
3- How many nanometers should the wavelength of the light source be?
4- If there is skin contact, the contacting surfaces must be biocompatible.
5- After the design is complete, the power should be verified with a
calibrated measuring instrument.

So in conclusion, yes, I think that the device can be easily developed, but
you should not take
legal responsibility for possible injuries and damages.

Best Regards & Good Luck

lopezwys ***@***.***>, 4 Şub 2023 Cmt, 16:49 tarihinde şunu
yazdı:

**Comment by lopezwys** [comment]
Thank you for your thorough response. I'd suspected that this project concerns something else, but i was hoping that you could nevertheless answer my questions, since the implemented solutions probably overlap, at least partially. Looks like i was correct. 
Thank you for listing the critical points, i'll definitely look into that. The legal aspect is also important, i'll keep that in mind. 
This project is still at very early stage, but your response helped to bring it one step closer to the actual development.
